packages/web-component/src/i18n/messages.eng.json

```diff
@@ -27,6 +28,10 @@
     "fra": "French",
     "spa": "Spanish",
     "close": "Close",
-    "annotations-tooltip": "Annotations"
+    "annotations-tooltip": "Annotations",
+    "edit-layer": "Rename annotation title",
+    "delete-layer": "Remove annotation title",
+    "save-layer": "Save annotation title",
+    "create-layer": "Create an annotation title"
```

Comment:
These are not intuitive to me. I would change all of them to "annotation layer" instead of "annotation title".